cypress/integration/test.js

```diff
@@ -40,37 +57,67 @@ describe('MovieDetail', () => {
     cy.get('section').should('have.class', 'movie-detail-page');
   });
 
-  // Here we should do unit testing for each element on the oage 
+  it('Should have two info articles', () => {
+    cy.get('article').should('have.length', 2);
+  });
+
+  it('Should have a backrop image', () => {
+    cy.get('.movie-detail-page div').should('have.class', 'backdrop-img');
+  });
 
-  // it('Should have a backrop image', () => {
-  //   cy.get('.movie-detail-page div').should('have.class', 'backdrop-img');
-  // });
+  it('Should have a title', () => {
+    cy.get('h2').should('contain', 'Mulan');
+  });
 
-  // it('Should have two info articles', () => {
-  //   cy.get('article').should('have.length', 2);
-  // });
+  it('Should have a release date', () => {
+    cy.get('p').should('contain', 'Release Date:');
+  });
 
-  // it('Should change urls when you click the home button', () => {
-  //   cy.get('button').click()
-  //     .get('section').should('have.class', '.poster-grid');
-  // }) this we are already testing so this would be a  duplicate
+  it('Should list genres', () => {
+    cy.get('p').should('contain', 'Genres:');
+  });
+
+  it('Should have a runtime', () => {
+    cy.get('p').should('contain', 'Runtime:');
+  });
+
+  it('Should have a budget', () => {
+    cy.get('p').should('contain', 'Budget:');
+  });
+
+  it('Should have a revenue', () => {
+    cy.get('p').should('contain', 'Revenue:');
+  });
+
+  it('Should have a tagline', () => {
+    //COME BACK TO ME
+    // cy.get('h3').should('contain', 'Revenue:')
+  });
+
+  it('Should have an overview', () => {
+    cy.get('p').should('have.class', 'overview');
+  });
+
+  it('Should have a rating', () => {
+    cy.get('p').should('contain', 'Freshness:');
+  });
 });
 
 describe('Error', () => {
-  it.skip('Should update the DOM to inform users of a 404', () => {
-    // dont know why but visit needs to go ahead of intercept to work 
-    cy.visit('http://localhost:3000/337401')
-    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/337401', {
-      ok: false,
-      status: 404,
-      statusText: 'Not Found'
-    })
+  it('Should update the DOM to inform users of a 404', () => {
+    cy.intercept({method: 'GET', url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'}, {
+      statusCode: 404
+    });
+    cy.visit('http://localhost:3000')
     .get('.error-status').should('have.text', '404 - Not Found')
     .get('.error-expl').should('have.text', 'What you\'re looking for isn\'t here. Double Check the url and try again.');
   });
-  
+
   it('Should update the DOM to inform users of a 500 error', () => {
-    cy.visit('http://localhost:3000/fakepath')
+    cy.intercept({method: 'GET', url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/337401'}, {
+      statusCode: 500
+    });
+    cy.visit('http://localhost:3000/337401')
```

Comment:
Great work, working through this intercept I know took awhile but glad it finally work!